ElectronClient/app/theme.js

```diff
@@ -386,10 +388,11 @@ function themeStyle(theme) {
 		textAreaLineHeight: Math.round(globalStyle.textAreaLineHeight * editorFontSize / 12),
 
 		// For WebView - must correspond to the properties above
-		htmlFontSize: `${Math.round(15 * zoomRatio)}px`,
+		htmlFontSize: `${Math.round(globalStyle.htmlFontSize * zoomRatio)}px`,
```

Comment:
I'd prefer if we don't introduce another theme variable.  `titleEditorFontSize` could simply be `fontSizes.htmlFontSize * 1.45`

Reply:
Done.  A little trickier than that, because `htmlFontSize` is a string with `px`, but easily handled.